prelude/diagnostic.js

```diff
@@ -4,14 +4,42 @@
 
 'use strict';
 
+function humanSize(bytes) {
+  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
+
+  if (bytes === 0) {
+    return 'n/a';
+  }
+
+  const i = Math.floor(Math.log(bytes) / Math.log(1024));
+
+  if (i === 0) {
+    return `${bytes} ${sizes[i]}`;
+  }
+
```

Comment:
Strictly speaking we should add a boundary checking here.
If someone passed a number of bytes that's larger than what 'TB' can represent, the function would try to access an index in the sizes array that doesn't exist, which would result in undefined.
But this is unlikely that someone will try to pack a file that is greater than a TB.

Reply:
> But this is unlikely that someone will try to pack a file that is greater than a TB.

Yeah I wouldn't expext that BTW js would just return `undefined` there, so something that would throw anyway